frontend/app-development/features/administration/components/DownloadRepoModal.tsx

```diff
@@ -33,7 +33,9 @@ export function DownloadRepoModal(props: IDownloadRepoModalProps) {
       >
         <div className={classes.modalContainer}>
           <h2>{t('administration.download_repo_heading')}</h2>
-          <p>{t('administration.download_repo_info')}</p>
+          <p>
+            <Trans i18nKey={'administration.download_repo_info'} />
```

Comment:
Nice catch that we need to use the `<Trans/>` component to ensure the HTML is rendered as HTML.

Anyway, should we try to avoid using HTML within our translation texts? 🤔 We might only use HTML tags for bold or emphasis texts. We should definitely avoid things like `<h1/>, <p/>` or other elements that have accessibility concerns. I do not say that you need to change anything about this within your PR. Im just writing my thoughts.

Reply:
I found an important message within the docs for i18n. 

"It does ONLY interpolation. It does not rerender on language change or load any translations needed. Check [useTranslation hook](https://github.com/latest/usetranslation-hook) or [withTranslation HOC](https://github.com/latest/withtranslation-hoc) for those cases." (https://react.i18next.com/latest/trans-component#important-note)

For me it sounds like when changing the language from Norwegian to English, the component `<Trans/> `does not re-render. This might cause bugs in the future when we enable the users to do switch between languages. 🤔 I think we should have this in mind and try to use the <Trans/> component in as few places as possible. Due to the documentation the good old t function should be enough.

Reply:
Atm this was the fix I came up with on the problem. We should cross that bridgwe when/if we introduce lanuage changes.

Reply:
Yes, and your fix is more than good enough and I approved it. Great work! 👏 I had no suggestions as to what could be changed. I thought it was smart to share what I found from the docs to share knowledge. 😅

Reply:
We should keep these comments to later, as we are getting married with i18next.

Reply:
Interesting note. Maybe we could use some hook to trigger a rerender of this particular component when the language changes? Just my initial thought.